core/templates/dev/head/services/explorationContextService.js

```diff
@@ -34,7 +34,7 @@ oppia.factory('explorationContextService', [
   function($window, PAGE_CONTEXT, EDITOR_TAB_CONTEXT) {
     var _pageContext = null;
     var _explorationId = null;
-
+    var isVideoPlayedList = {};
```

Comment:
Edit: maybe autoplayedVideosService, and autoplayedVideos/autoplayedVideosDict -- since that's really what we're using this for.